themes/vuetober/src/store/index.js

```diff
@@ -36,6 +37,9 @@ export default new Vuex.Store({
     addTeachers(state, teachers) {
       state.teachers = teachers;
     },
+    selectTeacher(state, teacher) {
```

Comment:
Mozno o cosi lepsi sposob ako trackovat ktory teacher je selected by bol, ze vzdy ked selectnes nejakeho teachera tak spravis nieco ako 
```
selectTeacher(state, selectedTeacherId) {
  state.teachers = state.teachers.map(teacher => {
    if (teacher.id === selectedTeacherId) {
		return {...teacher, isSelected: true};
	}
	return {...teacher, isSelected: false};
   })
}
```

Reply:
Vymyslel som to troška inak checkni najnovsi commit.